src/main/java/thrift/logic/parser/TagCommandParser.java

```diff
@@ -0,0 +1,50 @@
+package thrift.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static thrift.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import thrift.commons.core.index.Index;
+import thrift.logic.commands.TagCommand;
+import thrift.logic.parser.exceptions.ParseException;
+import thrift.model.tag.Tag;
+
+/**
+ * Parses input arguments and creates a new TagCommand object.
+ */
+public class TagCommandParser implements Parser<TagCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the TagCommand
+     * and returns an TagCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
```

Comment:
Do leave an empty line between description and parameter section.

Reply:
Added the empty line.